pkg/api/v1/helpers.go

```diff
@@ -326,6 +328,10 @@ func GetTolerationsFromPodAnnotations(annotations map[string]string) ([]Tolerati
 	return tolerations, nil
 }
 
+func GetPodTolerations(pod *Pod) ([]Toleration, error) {
```

Comment:
I'm always flummoxed by our api-machinery, is there a reason we dupe helpers -> v1/helpers?